package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Koa web app framework",
   "main": "lib/application.js",
   "scripts": {
-    "test": "make test"
+    "test": "jest",
+    "lint": "./node_modules/.bin/eslint benchmarks lib test"
```

Comment:
`npm run` prepends `./node_modules/.bin` to the `$PATH`, so it will use the local eslint even if it's also installed globally.

You can test it by adding `"tmp": "echo $PATH",` to the `package.json`.

[npm run docs](https://docs.npmjs.com/cli/run-script)